library/src/events.rs

```diff
@@ -16,16 +16,32 @@ impl Serialize for EventType {
     {
         serializer.serialize_str(match self {
             EventType::PatchInstallSuccess => "__patch_install__",
-            // EventType::PatchInstallFailure => "__patch_install_failure__",
+            EventType::PatchInstallFailure => "__patch_install_failure__",
         })
     }
 }
 
+impl<'de> Deserialize<'de> for EventType {
```

Comment:
Mostly for my learning: I know it's valid to give lifetime annotationss whatever name you'd like, but I most commonly see `'a`. Are there best practices around when/what to name them different things?

Reply:
It's just matching what the docs do for this specific class. 🤷  https://serde.rs/impl-deserialize.html